qemu/tests/sgx_basic.py

```diff
@@ -0,0 +1,47 @@
+from virttest import error_context
+from provider.sgx import SGXTestCapability
+from virttest import env_process
+from virttest.utils_misc import verify_dmesg
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx basic test:
+    1. Check host sgx capability
+    2. Boot sgx VM
+    3. Check sgx qmp cmd
+    4. Verify sgx flags and epc size
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sgx test", test.log.info)
+    timeout = float(params.get("login_timeout", 240))
+
+    sgx_cap = SGXTestCapability(test, params)
+    host_epc_size = sgx_cap.valid_host_sgx_cap()
+    params["host_epc_size"] = host_epc_size
+    if params.get("monitor_expect_nodes"):
+        sgx_cap.valid_host_numa_node()
+    params['start_vm'] = "yes"
+
+    vm_name = params["vms"]
+    env_process.preprocess_vm(test, params, env, vm_name)
```

Comment:
remove them, add vm.create() after line 33

Reply:
Thanks Zhenchao, try vm.create() after get_vm(), it works for me